test/cypress/integration/widgets/deployment_actions_spec.ts

```diff
@@ -1,5 +1,5 @@
-import type { LabelInputType } from '../../../../widgets/common/src/labels/types';
-import { isLabelModifiable } from '../../../../widgets/common/src/labels/common';
+import type { LabelInputType } from '../../../../app/widgets/common/labels/types';
+import { isLabelModifiable } from '../../../../app/widgets/common/labels/common';
```

Comment:
General:

Seeing that in `test/cypress/components/widgets/filters/RulesForm_spec.tsx` we use shorter paths in imports, can we have:
```suggestion
import type { LabelInputType } from 'app/widgets/common/labels/types';
import { isLabelModifiable } from 'app/widgets/common/labels/common';
```
here?

Applies to all other changes in `test/cypress/integration`.

Reply:
Yes, done.